src/warp.jl

```diff
@@ -70,39 +94,26 @@ julia> indices(imgr)
 (Base.OneTo(906),Base.OneTo(905))
 ```
 """
-function warp{T}(img::AbstractExtrapolation{T}, tform)
-    inds = autorange(img, tform)
+function warp{T}(img::AbstractExtrapolation{T}, tform, inds::Tuple = autorange(img, inv(tform)))
```

Comment:
we still need to use `inv` to compute the indices. But I changed it so that a user can provide them instead and thus avoid `inv`